src/controllers/segmentation.py

```diff
@@ -3,20 +3,33 @@
 import numpy as np
 import cv2
 
+
+class SegmentationClass:
```

Comment:
It would be better to use Enum type here https://docs.python.org/3/library/enum.html